src/chronometer.js

```diff
@@ -1,30 +1,39 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(printTimeCallback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime += 1;
+      printTimeCallback();
+    }, 1000);
   }
 
   getMinutes() {
-    // ... your code goes here
+    let minutes = Math.floor(this.currentTime / 60);
+    return minutes;
   }
 
   getSeconds() {
-    // ... your code goes here
+    let seconds = this.currentTime % 60;
+    return seconds;
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    if (value < 10) {
+      return `0${value}`;
+    }
+    return `${value}`;
   }
```

Comment:
Para transformar em string, vc pode pegar o value e executar o método .toString( )
Depois, teremos três returns...
se o value for identico a 0; retorna '00' dois zeros;
se o value for maior ou igual a 10; retorna o value transformado em string;
senao, retorna '0' em string + o value em string 
Faz o teste :)